api/proto/config/monitoring.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+package proto.config;
+
+option go_package = "github.com/spinnaker/kleat/api/client/config";
+
+import "metricstores/datadog.proto";
+import "metricstores/newrelic.proto";
+import "metricstores/prometheus.proto";
+import "metricstores/stackdriver.proto";
+
+// Configuration for the spinnaker-monitoring microservice.
+// The monitoring protos use snake_case for backwards compatibility with
+// Halyard-generated hal configs and the spinnaker-monitoring microservice.
+// All new protos should be added using camelCase for consistency with the
+// rest of the hal config.
```

Comment:
Let me know if you think this makes sense. Alternatively, we could require users to supply monitoring config in camelCase and document it as a breaking change, but then translate these fields to snake_case since that is how spinnaker-monitoring expects them.

Reply:
I think this makes sense, in the spirit of having `kleat` be as lightweight a translation layer as possible.